src/components/HorizontalStep/HorizontalStep.module.css

```diff
@@ -0,0 +1,59 @@
+/*------------------------------------*\
+    #HORIZONTAL STEP
+\*------------------------------------*/
+
+/**
+ * Horizontal Step displays the step text and associated decorative icon.
+ */
+.horizontal-step {
+  display: flex;
+  align-items: center;
+  gap: var(--eds-size-1);
+}
+
+/**
+ * Horizontal step variants
+ */
+.horizontal-step--complete {
+  color: var(--eds-theme-color-text-neutral-subtle);
+}
+.horizontal-step--active {
+  color: var(--eds-theme-color-text-neutral-strong);
+}
+.horizontal-step--incomplete {
+  color: var(--eds-theme-color-text-neutral-subtle);
+}
+
+/**
+ * Horizontal Step check circle icon
```

Comment:
Nit – I would avoid referencing the exact icon unless it's needed to understand the styling. These kinds of details in the design sometimes shift around and then the comments go stale and become confusing.

Reply:
Makes sense to me, how about `Horizontal Step Complete`?